pom.xml

```diff
@@ -73,13 +72,18 @@
     <version.org.gwtbootstrap3>0.9.4</version.org.gwtbootstrap3>
     <version.org.jboss.errai>4.3.3.Final</version.org.jboss.errai>
     <version.org.optaplanner>${project.version}</version.org.optaplanner>
+    <version.org.kie.soup>${project.version}</version.org.kie.soup>
     <version.org.slf4j>1.7.25</version.org.slf4j>
     <version.org.webjars.bower.org.patternfly>3.18.1</version.org.webjars.bower.org.patternfly>
     <version.org.webjars.Eonasdan-bootstrap-datetimepicker>4.17.47</version.org.webjars.Eonasdan-bootstrap-datetimepicker>
     <version.org.webjars.momentjs>2.22.2</version.org.webjars.momentjs>
     <version.org.wildfly-plugins.wildfly-maven-plugin>1.2.0.Final</version.org.wildfly-plugins.wildfly-maven-plugin>
```

Comment:
Do we still need this? I notice you removed the version from the wildfly-maven-plugin plugin but didn't remove the property. If it the same case as mockito, add a comment above it. Otherwise, remove the property.

Reply:
no, good catch, I removed that version property now.